.evergreen/ci_matrix_constants.js

```diff
@@ -1,8 +1,8 @@
 const MONGODB_VERSIONS = ['latest', 'rapid', '7.0', '6.0', '5.0', '4.4', '4.2', '4.0', '3.6'];
 const versions = [
-  { codeName: 'gallium', versionNumber: 16 },
-  { codeName: 'hydrogen', versionNumber: 18 },
-  { codeName: 'iron', versionNumber: 20 }
+  { codeName: 'gallium', versionNumber: 16, npmVersion: 9 },
+  { codeName: 'hydrogen', versionNumber: 18, npmVersion: 9 },
```

Comment:
https://www.npmjs.com/package/npm

npm 10 should be fine as long as the 18 version is above >= 18.17.0, it should only be our performance node version which is 18.16 (😞) that needs the lower npm version